### ISSUE TITLE
Some LaTeX math symbols don't work

### ISSUE THREAD

**Issue by eurunuela** [myst-templates/lapreprint#1]
I'm building the PDF of my manuscript with this template but some LaTeX math symbols are shown as empty squares; e.g., the letter Omega does not show.

<img width="736" alt="CleanShot 2023-01-26 at 12 00 02@2x" src="https://user-images.githubusercontent.com/13706448/214830199-f57a5d6b-e9ec-44b3-9290-5faa22c8a83e.png">

**Comment by rowanc1** [comment]
Yes, I saw this as well -- this is my favourite template! I think that it might be a compiler or font thing.

I had to hard-fork this repo (on Monday) to get the actions to work (github annoyingly doesn't enable forks to run actions), but my intention is to contribute it back upstream so that the myst community isn't the only one caring about/helping with this!!

If you have a chance to debug this more, I would be curious if you export using `tex` and then compile that tex in the way you normally would, and see if that is still an issue. If it isn't then we probably need to tweak our engine that we use (xelatex) or fix an issue in the template, but would much prefer not to hard fork it, so I will do some work to contribute back up the original repository.

**Comment by eurunuela** [comment]
I can try to export to `tex` and see how that goes.

In any case, if you're going to contribute to this template, may I ask that you remove the extra side margin past page one? It feels like a waste of space. I would 100% use this template if it weren't for these two issues.

Edit: Okay, it looks like I have the following warning related to the font.

```shell
LaTeX Font Warning: Font shape `U/ifsym/b/n' undefined
(Font)              using `U/ifsym/m/n' instead on input line 121.
```

But the PDF looks good and the Omega is there.

The only error I get is for an undefined equation reference that actually gets rendered on the PDF, weird.

**Comment by rowanc1** [comment]
There is an issue about the margin here, which maybe helpful to comment on? As well as some workarounds. 

https://github.com/roaldarbol/LaPreprint-class/issues/4

Having easy access to toggle that on/off would be nice. If you find a way to do it in Tex (there are suggestions in the issue) maybe we could incorporate that into the MyST template as an option.

Annoying about the font. I will try to look into that soon. We use xetex to protect against some Unicode in myst, but maybe we can do a better job prefiltering and change the engine. There is also likely a way to config the font....!?!

**Comment by roaldarbol** [comment]
Hi there, just saw this issue! I am moving the document class into [roaldarbol/LaPreprint-class](https://github.com/roaldarbol/LaPreprint-class), so any issues and solutions would be more than welcome there. :-) 

As for the omega, I don't think it's an issue stemming from LaPreprint, at least I can verify that I do get an omega symbol with `\Omega`.

I haven't found a solution for getting rid of the margin space - any help would be appreciated on roaldarbol/LaPreprint-class#4. 

@rowanc1, I hope to bring the template to CTAN during the summer - I hope that might help to make it easier to implement in MyST, would be really happy to see it in use there!

**Comment by rowanc1** [comment]
Yes, absolutely we can rebase our small fork of the repo here! I am glad you found this repo!! :)

Folks using MyST really enjoy this template! Let me know if there is a better place to look out for updates or coordinate on this.

**Comment by roaldarbol** [comment]
The [roaldarbol/LaPreprint-class](https://github.com/roaldarbol/LaPreprint-class) will be the place where changes on the `class` will happen!
